hedera-node/src/main/java/com/hedera/services/txns/token/TokenAssociateTransitionLogic.java

```diff
@@ -50,53 +50,25 @@
 public class TokenAssociateTransitionLogic implements TransitionLogic {
 	private final Function<TransactionBody, ResponseCodeEnum> SEMANTIC_CHECK = this::validate;
 
-	private final AccountStore accountStore;
-	private final TypedTokenStore tokenStore;
+
 	private final TransactionContext txnCtx;
-	private final GlobalDynamicProperties dynamicProperties;
+	private final AssociateLogic associateLogic;
 
+	// TODO: Global dynamic properties are not being used in the transition logic itself and is being moved to the
+	//  	 Associate logic. Confirm, that this solution is appropriate.
 	@Inject
```

Comment:
Seems fine to me, I think we can remove this TODO

Reply:
Done.